blueox/contrib/celery/celery_signals.py

```diff
@@ -43,13 +42,13 @@ def on_task_sent(**kwargs):
 
 @signals.worker_process_init.connect
 def on_worker_process_init(**kwargs):
-    host = getattr(settings, 'BLUEOX_HOST', '127.0.0.1')
-    port = getattr(settings, 'BLUEOX_PORT', 3514)
-    blueox.configure(host, port)
-
-    handler = blueox.LogHandler()
-    handler.setLevel(logging.INFO)
-    logging.getLogger('').addHandler(handler)
+    if hasattr(settings, 'BLUEOX_HOST'):
+        if settings.BLUEOX_HOST:
+            blueox.default_configure(settings.BLUEOX_HOST)
+        else:
```

Comment:
ah ok makes sense. also what about using kwargs so people know what is being set to `None`?